app/backend/wells/models.py

```diff
@@ -1363,7 +1382,7 @@ class ActivitySubmission(AuditModelStructure):
                     'depth which is the total depth at which the well was drilled. The finished depth is '
                     'represented in units of feet bgl (below ground level).'))
     final_casing_stick_up = models.DecimalField(
-        max_digits=6, decimal_places=3, blank=True, null=True, verbose_name='Final Casing Stick Up')
+        max_digits=5, decimal_places=3, blank=True, null=True, verbose_name='Final Casing Stick Up')
```

Comment:
I think this is a merge conflict? Is there a reason you're taking it back down to 5?
```suggestion
        max_digits=6, decimal_places=3, blank=True, null=True, verbose_name='Final Casing Stick Up')
```